src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -565,6 +566,168 @@ JSON example:
+| VascularCrushing                  | RECOMMENDED                                                                                                                     | [boolean][]                             | Boolean indicating if Vascular Crushing is used. Corresponds to DICOM Tag 0018,9259 `ASL Crusher Flag`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| AcquisitionVoxelSize              | RECOMMENDED                                                                                                                     | [array][] of [numbers][]                | An array of numbers with a length of 3, in millimeters. This parameter denotes the original acquisition voxel size, excluding any inter-slice gaps and before any interpolation or resampling within reconstruction or image processing. Any point spread function effects, for example due to T2-blurring, that would decrease the effective resolution are not considered here.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
@effigies While I am working on common fields: should this one become a common field as well?